graphql_compiler/schema_transformation/utils.py

```diff
@@ -33,7 +34,33 @@ class InvalidTypeNameError(SchemaTransformError):
 
 
 class SchemaNameConflictError(SchemaTransformError):
-    """Raised when renaming types or fields cause name conflicts."""
+    """Raised when renaming or merging types or fields cause name conflicts."""
+
+
+_alphanumeric_and_underscore = frozenset(string.ascii_letters + string.digits + '_')
```

Comment:
We have something almost identical [here](https://github.com/kensho-technologies/graphql-compiler/blob/master/graphql_compiler/compiler/helpers.py#L21)-- that version is python2 and 3 compatible too.

Might be a bit odd importing something from `..compiler`, so maybe move that definition into some higher-level directory?

Reply:
The `six.text_type` part might be good to grab from there, but redefining this here as well is not awful -- it's a one-liner, and there's no semantic connection to the other code that would be aided by deduplicating these two lines.